### DISCUSSION TITLE
I can't receive any SMS verification. I've tried so many times yet still quiet

### DISCUSSION THREAD

**Discussion by finnwasabi** [vercel/community#4666] [Account Issue]
### Question

Please help me because i really need Vercel ASAP

**Comment by amyegan** [comment]
Hi @finnwasabi. We can't safely discuss account info here in public. Did you already [reach out to the Customer Success team](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)? If you have a case number, I can check on the ticket for you 🙏
